doc/multiple-output.xml

```diff
@@ -270,8 +270,8 @@
      </listitem>
     </varlistentry>
     <varlistentry>
-     <term><varname>
-        $outputInfo</varname>
+     <term>
+      <varname> $outputInfo</varname>
      </term>
      <listitem>
       <para>
```

Comment:
__note__ this is adding an extra space after  `<varname>`, maybe it is fixed in a future commit

Reply:
This isn't adding an extra space after `<varname>`, but condensing the run of spacing into one.